lib/server.ts

```diff
@@ -127,11 +139,11 @@ class ThingsHandler extends BaseHandler {
    * @param {Object} req The request object
    * @param {Object} res The response object
    */
-  get(req, res) {
+  get(req: any, res: core.Response): void {
     const wsHref = `${req.secure ? 'wss' : 'ws'}://${req.headers.host}`;
     res.json(
       this.things.getThings().map((thing) => {
-        const description = thing.asThingDescription();
+        const description = thing.asThingDescription() as any;
```

Comment:
Why not `as Thing.ThingDescription`?

Reply:
The reason I used any was that `Thing.ThingDescription` does not contain properties for `base`, `securityDefinitions`, `security`. I didn't know if they should be added. Should I add them? Or should I use any? or should I create a subtype of `Thing.ThingDescription`?

Reply:
Those additional fields are defined in the spec and will be present when viewing the thing description via the REST API, so they should be part of `Thing.ThingDescription`. Obviously, make them optional if they're not added until this point.